### PR TITLE
add quest, Servlet, Service

### PR DESCRIPTION
**Pull request by Seragors**

### CODE REVIEW

File: src/main/java/service/capitanService.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package service;
+
+public class capitanService {
```
Comment (Volodymyr95) [inline src/main/java/service/capitanService.java]: why class name from small letter???

File: src/main/java/service/capitanService.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    public String call(String choice) {
+        String accepted  = null;
+        if ("accept".equals(choice)) {
```
Comment (Volodymyr95) [inline src/main/java/service/capitanService.java]: make accept constant

File: src/main/java/service/capitanService.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+            accepted = ("whoAreYou.jsp");
```
Comment (Volodymyr95) [inline src/main/java/service/capitanService.java]: make whoAreYou constant

File: src/main/java/service/incrementGames.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package service;
+
+import javax.servlet.http.HttpSession;
+
+public class incrementGames {
```
Comment (Volodymyr95) [inline src/main/java/service/incrementGames.java]: is it servlet?

**Comment by Volodymyr95** [comment]
in Java All classes starts from Big letter, not Smal!

File: src/main/java/service/CapitanService.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package service;
+
+public class CapitanService {
+    private final String accept = "accept";
+    private final String reject = "reject";
+    private final String acceptIndex = "whoAreYou.jsp";
+    private final String rejectIndex = "lose.jsp";
+    private String accepted;
```
Comment (Volodymyr95) [inline src/main/java/service/CapitanService.java]: create it inside method, and rename to result

File: src/main/java/service/GameService.java [hunk 6]
```diff
@@ -0,0 +1,18 @@
+package service;
+
+public class GameService {
+    private final String accept = "accept";
```
Comment (Volodymyr95) [inline src/main/java/service/GameService.java]: u are using there constants in few classes, make it public and create separate constant class for it

File: src/main/java/servlet/IncrementGames.java [hunk 7]
```diff
@@ -0,0 +1,18 @@
+package servlet;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpSession;
+
+public class IncrementGames extends HttpServlet {
```
Comment (Volodymyr95) [inline src/main/java/servlet/IncrementGames.java]: it's a servlet?

File: src/main/java/servlet/LoseServlet.java [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+import service.LoseService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/")
```
Comment (Volodymyr95) [inline src/main/java/servlet/LoseServlet.java]: why / ??

File: src/main/java/servlet/WhoAreYouServlet.java [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+import service.WhoAreYouService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/whoAreYou")
```
Comment (Volodymyr95) [inline src/main/java/servlet/WhoAreYouServlet.java]: who-are -you

File: src/main/java/servlet/IncrementGames.java (hunk 7 above)
Comment (Seragors) [reply src/main/java/servlet/IncrementGames.java]: Yes, this is a servlet, I moved it to the servlet where the session is GameServlet

**Review by Volodymyr95 — APPROVED** [review]

**Pull request merged**